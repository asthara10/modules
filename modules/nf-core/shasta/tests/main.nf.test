nextflow_process {

    name "Test Process SHASTA"
    script "../main.nf"
    config "./nextflow.config"
    process "SHASTA"

    tag "modules"
    tag "modules_nfcore"
    tag "shasta"

    test("test_shasta") {


        when {
            process {
                """
                
                input[0] = [ 
        [ id:'test', model:'Nanopore-Dec2019' ], // meta map
        [ file(params.test_data['candidatus_portiera_aleyrodidarum']['nanopore']['test_fastq_gz']) ],
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
