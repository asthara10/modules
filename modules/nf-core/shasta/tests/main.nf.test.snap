{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T10:28:28.538505"
    }
}